ui/src/FindBar.tsx

```diff
@@ -131,22 +219,26 @@ export class FindBar extends React.PureComponent<{}, FindBarState> {
       return null;
     }
 
-    if (matchCount > MATCH_COUNT_LIMIT) {
+    if ((matchCount || 0) > MATCH_COUNT_LIMIT) {
```

Comment:
Is this fix just to satisfy type checking?

Reply:
yes :(